src/applications/mhv-secure-messaging/tests/e2e/fixtures/generalResponses/featureToggles.json

```diff
@@ -846,6 +846,14 @@
         "name": "form5495_confirmation_email",
         "value": true
       },
+      {
+        "name": "form214138",
+        "value": true
+      },
+      {
+        "name": "form214138",
+        "value": true
+      },
```

Comment:
May I please get more context on how and why these feature flags are impacting Secure Messaging ?

Reply:
Oops! Sorry about that, wrong directory. I'll remove these changes ASAP.